### DISCUSSION TITLE
php firebaser

### DISCUSSION THREAD

**Discussion by basitkhna** [kreait/firebase-php#615] [General]
Fatal error: Uncaught GuzzleHttp\Exception\ClientException: Client error: `POST https://www.googleapis.com/identitytoolkit/v3/relyingparty/signupNewUser` resulted in a `400 Bad Request` response: (truncated...) in C:\xampp\htdocs\php_firebase\vendor\guzzlehttp\guzzle\src\Exception\RequestException.php:113 Stack trace: #0 C:\xampp\htdocs\php_firebase\vendor\guzzlehttp\guzzle\src\Middleware.php(69): GuzzleHttp\Exception\RequestException::create(Object(GuzzleHttp\Psr7\Request), Object(GuzzleHttp\Psr7\Response), NULL, Array, NULL) #1 C:\xampp\htdocs\php_firebase\vendor\guzzlehttp\promises\src\Promise.php(204): GuzzleHttp\Middleware::GuzzleHttp\{closure}(Object(GuzzleHttp\Psr7\Response)) #2 C:\xampp\htdocs\php_firebase\vendor\guzzlehttp\promises\src\Promise.php(153): GuzzleHttp\Promise\Promise::callHandler(1, Object(GuzzleHttp\Psr7\Response), NULL) #3 C:\xampp\htdocs\php_firebase\vendor\guzzlehttp\promises\src\TaskQueue.php(48): GuzzleHttp\Promise\Promise::GuzzleHttp\Promise\{closure}() #4 C:\xampp\htdocs\php_firebase\vendor in C:\xampp\htdocs\php_firebase\vendor\kreait\firebase-php\src\Firebase\Exception\AuthApiExceptionConverter.php on line 75

**Comment by basitkhna** [comment]
<?php
session_start();
require_once('connection.php');
if(isset($_POST['submit']))
{
    
    $email=$_POST['email'];
    $password=$_POST['password'];
    $userProperties = [
        'email' => $email,      
        'password' => $password,
    ];    
    $createdUser = $auth->createUser($userProperties);
    if($createdUser)
    {
        $_SESSION['status']="user Creaed Successfully";
          header("location:sighnup.php");
    }
    else
    {
        $_SESSION['status']="User not created Successfully";
           header9("location:sighnup.php");
    }
}
?>

**Reply by basitkhna** [reply]
how we can fix this issue please help me

**Reply by jeromegamez** [reply]
When want others to take the time to help you with a problem, please take the time to write a problem description that is easier to understand. For example, put it in one post instead of three, and you can use Markdown to format code (also with syntax highlighting), and please add what you want to achieve. You can edit your own posts, too.

That being said: your code in the second post is incomplete and cannot be used to reproduce it (= I can not copy and paste it to see if it works for me).
